src/js/module/LinkDialog.js

```diff
@@ -67,6 +67,9 @@ export default class LinkDialog {
   }
 
   checkLinkUrl(linkUrl) {
+    if (this.options.onCreateLink) {
```

Comment:
I use the same option used in `Editor.js` module to be consistent

https://github.com/summernote/summernote/blob/main/src/js/module/Editor.js#L216